### DISCUSSION TITLE
Not found Extension Development debugger

### DISCUSSION THREAD

**Discussion by datlechin** [microsoft/vscode-discussions#110] [Extension Development QnA] [answered]
Hi VSCoder,
I'm newbie here, i'm creating new vscode language package, but I have problem here.

I press `F5` but there no have option called `VS Code Extension Development`

![image](https://user-images.githubusercontent.com/56961917/186575912-0fe32085-672d-4d94-812c-b0178fa9efbd.png)

**Comment by isidorn** [comment, marked as answer]
@datlechin I suggest to start from an extension template https://code.visualstudio.com/api/get-started/your-first-extension
You are basically missing a `launch.json` which specifies to VS Code how to debug your extension. I suggest to copy the one from the sample/.vscode/launch.json and then modify to suite your needs. Hope that helps
